content/docs/concepts/authorization/v1.x/_index.md

```diff
@@ -29,9 +29,11 @@ The following diagram shows a high-level overview of Container Storage Modules f
 **NOTE:** PowerScale OneFS implements its own form of Role-Based Access Control (RBAC). Authorization does not enforce any role-based restrictions for PowerScale. To configure RBAC for PowerScale, refer to the PowerScale OneFS [documentation](https://www.dell.com/support/home/en-us/product-support/product/isilon-onefs/docs).
 
 ## Authorization Components Support Matrix
-Authorization consists of 2 components - The authorization sidecar, bundled with the driver, communicates with the Authorization proxy server to validate access to Storage platforms. The authorization sidecar is backward compatible with older Authorization proxy server versions. However, it is highly recommended to have the Authorization proxy server and sidecar installed from the same release of Container Storage Modules.
+Authorization consists of two main components - the Authorization Sidecar, bundled with the driver, which forwards requests to the Authorization proxy server, and the Authorization Proxy Server, which validates access to Storage platforms.
 
-**NOTE:** If the deployed CSI driver has a number of controller pods equal to the number of schedule nodes in your cluster, Authorization may not be able to inject properly into the driver's controller pod.
+We highly recommend that customers install or upgrade the Authorization proxy server and sidecar from the same release of CSM. This practice helps ensure optimal performance and compatibility, avoiding potential issues from version discrepancies.
```

Comment:
This needs to be definitive. "We highly recommend..." is not clear and left to interpretation.

Reply:
Fixed